apps/dashboard/src/contract-ui/tabs/listings/components/list-form.tsx

```diff
@@ -50,6 +50,7 @@ import type {
 } from "thirdweb/extensions/marketplace";
 import { useActiveAccount, useSendAndConfirmTransaction } from "thirdweb/react";
 import {
+  Card,
```

Comment:
do not add new imports from `tw-components` - we are removing this, there should be a `card` component in shadcn components we already use